src/v/cluster/tx_gateway_frontend.h

```diff
@@ -26,7 +30,15 @@
 
 namespace cluster {
 
-class tx_gateway_frontend {
+struct tx_coordinator_address {
+    kafka::error_code error;
+    model::node_id node;
+    ss::sstring host;
+    int32_t port;
```

Comment:
how are host/port used? we have structures like `unresolved_address` that store a host and port. seastar also has some types that might be appropriate depending on the use case.

Reply:
it's a rudiment, this structure is never used, removing it altogether